foxglove/cmd/root.go

```diff
@@ -63,6 +63,26 @@ func listDevicesAutocompletionFunc(
 	}
 }
 
+func listDevicesByNameAutocompletionFunc(
+	baseURL string,
+	clientID string,
+	token string,
+	userAgent string,
+) func(*cobra.Command, []string, string) ([]string, cobra.ShellCompDirective) {
+	return func(cmd *cobra.Command, args []string, toComplete string) ([]string, cobra.ShellCompDirective) {
+		client := console.NewRemoteFoxgloveClient(baseURL, clientID, token, userAgent)
```

Comment:
I suspect this isn't picking up baseURL as expected, and fails silently in this case. Not a big deal, but if this is the case I wonder if we should skip the autocomplete if baseURL is specified.

Reply:
hmm, good point. You're probably right. I'll play around with this.

Reply:
resolved with the approach discussed in slack